packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantImage/_tests/RestaurantImage.test.js

```diff
@@ -1,6 +1,13 @@
-import { mount } from '@vue/test-utils';
+import { mount, config } from '@vue/test-utils';
 import RestaurantImage from '../RestaurantImage.vue';
 
+// Allows for the testing of CSS module classes
+config.mocks.$style = new Proxy({}, {
+    get (_, name) {
+        return name;
+    }
+});
+
```

Comment:
Makes sense, happy to do that 😄

Reply:
I've found this 
https://github.com/justeat/fozzie-components/blob/caf032fb10f3dc5475c505cb812799e6b80c82b8/packages/components/molecules/f-media-element/src/components/_tests/MediaElement.test.js#L75

Reply:
I'm assuming that by making a global change, I'd have to do a version bump and update the change log for the root project.

Reply:
Only just seen this (I think it must have got lost in my emails when I was on holiday at the time).

I think this could potentially be useful – may be worth a chat on Slack about it (not sure if it being closed means it's not relevant anymore though?)

Reply:
What's the best way to pick this back up? Shall we have a hangout to discuss it when you're free later in the week?